pages.es/common/[.md

```diff
@@ -0,0 +1,25 @@
+# [
+
+> Evalúa una condicion.
+> Retorna 0 si la condición se cumple, 1 si esta no se cumple.
+> Más información: <https://www.gnu.org/software/coreutils/test>.
+
+- Prueba si una variable dada es igual a una cadena dada:
+
+`[ "{{MI_VARIABLE}}" == "{{/bin/zsh}}" ]`
```

Comment:
```suggestion
`[ "{{$MI_VARIABLE}}" == "{{/bin/zsh}}" ]`
```

Reply:
I don't really like the use of "My" for example values.  Can we change this, and the English version, to simply `VARIABLE` (assuming that's correct Spanish ;)